.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.1.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
+    -   id: check-yaml
+    -   id: check-added-large-files
+    -   id: check-merge-conflict
+    -   id: debug-statements
+    -   id: mixed-line-ending
+    -   id: trailing-whitespace
+-   repo: https://github.com/asottile/pyupgrade
+    rev: v3.2.0
+    hooks:
+      - id: pyupgrade
+        args: ["--py37-plus"]
```

Comment:
Just like the Django version, this is the minimum supported python version. Python code in a commit is then optimised to match the features in this python version.

Reply:
Alright Mark, thanks for clarifying